### PR TITLE
Allow using PIL with predict functions

### PR DESCRIPTION
**Pull request by johnbradley**
Fixes original issue from #54

### CODE REVIEW

**Review by hlapp — APPROVED** [review]
Looks good; the one thing that strikes me as a little awkward is passing PIL image objects to a parameter named `image_paths`.

Perhaps it's worth considering to rename this parameter to `images` (it seems just as OK to me to pass a list of image file paths to `images` as passing a list of image objects). Or have both parameters (if we want to aim for backwards compatibility), where passing both is an error.

Anyway, I don't feel too strong about this, so choosing to approve.

**Review by thompsonmj — APPROVED** [review]
Just a small typo, and I second @hlapp's comment on the optional change to the `image_paths` argument.

**Review by thompsonmj — APPROVED** [review]
Looks good!

File: tests/test_predict.py [hunk 1]
```diff
@@ -138,9 +159,29 @@ def test_predict_with_bins_bad_values(self):
...
         classifier = TreeOfLifeClassifier(device='cpu')
         self.assertEqual(classifier.model_str, 'hf-hub:imageomics/bioclip')
-        features = classifier.create_image_features_for_path(EXAMPLE_CAT_IMAGE, normalize=False)
+        features = classifier.create_image_features_for_image(EXAMPLE_CAT_IMAGE, normalize=False)
+        self.assertEqual(features.shape, torch.Size([512]))
+
+    def test_get_image_features_pil(self):
+        classifier = TreeOfLifeClassifier(device='cpu')
+        self.assertEqual(classifier.model_str, 'hf-hub:imageomics/bioclip')
+        features = classifier.create_image_features_for_image(EXAMPLE_CAT_IMAGE, normalize=False)
```
Comment (egrace479) [inline tests/test_predict.py:186]:
```suggestion
        features = classifier.create_image_features_for_image(PIL.Image.open(EXAMPLE_CAT_IMAGE), normalize=False)
```
I believe this one is meant to test the output using `PIL`?

**Review by egrace479 — APPROVED** [review]
LGTM!

**Pull request merged**